### DISCUSSION TITLE
Separate organization for Snakeware

### DISCUSSION THREAD

**Discussion by TechStudent10** [joshiemoore/snakeware#168] [General]
On GitHub, you can create organizations. I think that it makes sense for Snakeware to have its own. In the organization, you could have different repositories for stuff like `snake_apps`, `snake_games` and even a website.

**Comment by TechStudent10** [comment]
@joshiemoore I have decided I will create the Unofficial Snakeware Organization.

**Reply by TechStudent10** [reply]
@Snakeware-Unofficial has been created.

**Reply by TechStudent10** [reply]
After further thought, I might not do that.

**Reply by ghost** [reply]
I would be happy if there was an organization, so that could help
